modules/luci-mod-notification/htdocs/luci-static/resources/preload/notification.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+'require ui';
+'require rpc';
+'require poll';
+'require baseclass';
+
+var callNotificationRemove = rpc.declare({
+	object: 'system.notification',
+	method: 'remove',
+	params: [ 'uuid' ],
+	expect: { '': {} }
+});
+
+var callNotificationList = rpc.declare({
+	object: 'system.notification',
+	method: 'list',
+	expect: { messages: [] }
+});
+
+var callNotificationFlush = rpc.declare({
+	object: 'system.notification',
+	method: 'flush',
+	expect: { '': {} }
+});
+
+var info = {};
+
+return baseclass.extend({
+	__init__: function() {
+		this.updateNotification();
+		poll.add(L.bind(this.updateNotification, this), 5);
+	},
+
+	flushNotification: function(uuid, ev) {
+		return callNotificationFlush(uuid).then(L.bind(function(result) {
+				ui.hideModal();
+				ui.hideIndicator('notification');
+			}, this));
+	},
+
+	removeNotification: function(uuid, ev) {
+		return callNotificationRemove(uuid).then(L.bind(function(result) {
+				for( var i = 0; i < info.messages.length; i++) {
+					if ( info.messages[i].uuid === uuid) {
+						info.messages.splice(i, 1);
+					}
+				}
+				this.updateList();
+			}, this));
+	},
+
+	updateList: function() {
+		var rows = [];
+		for (var i = 0; i < info.messages.length; i++) {
+			var message = info.messages[i];
+			var time = new Date(message.timestamp*1000);
+
+			var date ='%02d/%02d/%04d %02d:%02d:%02d'.format(
+				time.getDate(),
+				time.getMonth() + 1,
+				time.getFullYear(),
+				time.getHours(),
+				time.getMinutes(),
+				time.getSeconds()
```

Comment:
why like this? why not letting the browser show it in browser locale? (time.toLocaleString()) (And if we're going to ignore browser locale, I'd at least vote for iso8601...)

Reply:
Have now used the nativ javascript implementation.
Thanks for the hint